### PR TITLE
planner_selector in migration guide

### PR DESCRIPTION
**Pull request by pabloinigoblasco**
Simple documentation to include the planner_selector on the migration guide.

### CODE REVIEW

File: plugins/index.rst [hunk 1]
```diff
@@ -244,7 +244,8 @@ Behavior Tree Nodes
 +--------------------------------------------+---------------------+------------------------------------------+
 | `Truncate Path`_                           | Francisco Martín    | Modifies a path making it shorter        |
 +--------------------------------------------+---------------------+------------------------------------------+
-
+| `Planner Selector`_                        | Pablo Iñigo Blasco  | Selects the global planner               |
```
Comment (SteveMacenski) [inline plugins/index.rst:247]: "based on a topic input, otherwises uses a default planner id"

File: migration/Foxy.rst [hunk 2]
```diff
@@ -199,3 +199,10 @@ These plugins are set as default in the ``nav2_bt_navigator`` but may be overrid
 Original GitHub tickets:
 
 - `SingleTrigger <https://github.com/ros-planning/navigation2/pull/2236>`_
+
```
Comment (SteveMacenski) [inline migration/Foxy.rst:202]: Just above this is the `SingleTrigger` added BT node section, you can remove this new section and just add this to the list after SingleTrigger with a link to the PR

File: configuration/packages/bt-plugins/actions/PlannerSelector.rst [hunk 3]
```diff
@@ -0,0 +1,56 @@
+.. _bt_planner_selector_node:
```
Comment (SteveMacenski) [inline configuration/packages/bt-plugins/actions/PlannerSelector.rst:1]: There's no link to this page anywhere, please add this page to the list https://692-240847415-gh.circle-artifacts.com/0/html/configuration/packages/configuring-bt-xml.html on this page under Action Plugins to have this page then properly linked

File: configuration/packages/bt-plugins/actions/PlannerSelector.rst [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+PlannerSelector
+==============
+
+It is used to select the planner that will be used by the planner server. It subscribes to the _planner_selector_ topic to recive command msgs with the name of the planner to be used. It is commonly used before of the ComputePathToPoseAction. The selected_planner output port is passed to planner_id input port of the ComputePathToPoseAction.
```
Comment (SteveMacenski) [inline configuration/packages/bt-plugins/actions/PlannerSelector.rst:6]:
```suggestion
It is used to select the planner that will be used by the planner server. It subscribes to the ``planner_selector`` topic to receive command messages with the name of the planner to be used. It is commonly used before of the ComputePathToPoseAction. The ``selected_planner`` output port is passed to ``planner_id`` input port of the ComputePathToPoseAction. If none is provided on the topic, the ``default_planner`` is used.
```

**Review by SteveMacenski — CHANGES_REQUESTED** [review]

**Comment by SteveMacenski** [comment]
https://github.com/ros-planning/navigation.ros.org/blob/master/configuration/packages/configuring-bt-xml.rst Last thing missing is your rst file has to be added to the Action list here so that it can be accessed

File: configuration/packages/bt-plugins/actions/PlannerSelector.rst (hunk 3 above)
Comment (pabloinigoblasco) [reply configuration/packages/bt-plugins/actions/PlannerSelector.rst:1]: This should be already done:
https://github.com/ros-planning/navigation.ros.org/pull/152/commits/3bc2462254a2e33bb2e1ecc959b546d9275edc53

**Review by SteveMacenski — APPROVED** [review]

**Pull request merged**